.github/workflows/vs17-cxx20.yml

```diff
@@ -21,4 +21,4 @@ jobs:
           cd build &&
           cmake  ${{matrix.cxx}} ${{matrix.arch}}  -DCMAKE_CXX_STANDARD=20 -DFASTFLOAT_TEST=ON -DCMAKE_INSTALL_PREFIX:PATH=destination ..  &&
           cmake --build .  --verbose &&
-          ctest --output-on-failure
+          ctest --output-on-failure -C Debug
```

Comment:
This seems to effectively disable our tests. Please see CI outputs.

Reply:
Indeed, thanks for the catch. Fix incoming.